attributes/default.rb

```diff
@@ -36,7 +36,7 @@
 default['cluster']['instance_types_data_path'] = "#{node['cluster']['shared_dir']}/instance-types-data.json"
 default['cluster']['computefleet_status_path'] = "#{node['cluster']['shared_dir']}/computefleet-status.json"
 default['cluster']['shared_storages_mapping_path'] = "/etc/parallelcluster/shared_storages_data.yaml"
-default['cluster']['update_shared_storages_mapping_path'] = "/etc/parallelcluster/update_shared_storages_data.yaml"
+default['cluster']['prvious_shared_storages_mapping_path'] = "/etc/parallelcluster/previous_shared_storages_data.yaml"
```

Comment:
Typo: prvious => previous

In other places within this PR, we reference with the correct key (not typo), so it's not clear to me how did it work.

Furthermore, I suggest to name this file as `pre_update_shared_storage_mapping` rather than `previous_shared_storage_mapping` as it would be more self-explanatory.

Reply:
Thanks for fixing it.